### DISCUSSION TITLE
CF dynamics

### DISCUSSION THREAD

**Discussion by uhvardhan** [USC-ACTLab/crazyswarm#605] [Q&A] [answered]
Hello, 

I was wondering where to start if I am to implement my own quadrotor dynamics. Specifically, I am interested in using the dynamics mentioned in this paper:
https://arxiv.org/pdf/2201.10509.pdf

Thank you.
Harsh

**Comment by whoenig** [comment, marked as answer]
The simulator currently only visualizes waypoints and does not consider any dynamics. For physics-based simulation, you can consider RotorS or CrazyS.

**Reply by uhvardhan** [reply]
Thank you.
